smokes/e2e/buildsnavigation.scenarios.coffee

```diff
@@ -10,16 +10,18 @@ homePage = require('./pages/home.coffee')
 describe 'previousnextlink', () ->
     force = null
     builder = null
+    home = null
 
-    beforeEach(() ->
+    beforeEach () ->
         builder = new builderPage('runtests', 'force')
         force =  new forcePage()
-    )
+        home = new homePage()
+        builder.goDefault()
```

Comment:
please remove those builder.goDefault() and put home.go() inside the renamed home.getLoginUser()